opm/simulators/wells/WellProdIndexCalculator.hpp

```diff
@@ -0,0 +1,56 @@
+/*
+  Copyright 2020 Equinor ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_WELLPRODINDEXCALCULATOR_HEADER_INCLUDED
+#define OPM_WELLPRODINDEXCALCULATOR_HEADER_INCLUDED
+
+#include <cstddef>
+#include <vector>
+
+namespace Opm {
+    class Well;
+} // namespace Opm
+
+namespace Opm {
+    class WellProdIndexCalculator
```

Comment:
Please add some documentation explaining the basic idea of the class, the fact that it contains only static well information etc.